src/raftstore/store/fsm/apply.rs

```diff
@@ -609,6 +620,7 @@ pub struct ApplyDelegate {
     /// If the delegate should be stopped from polling.
     /// A delegate can be stopped in conf change, merge or requested by destroy message.
     stopped: bool,
+    written: bool,
```

Comment:
I think it's better to use a counter to adjust configuration conveniently.
The write times at a time should be limited, but two times may be too strict?
It may lead to a follower who falls behind its leader a lot uses much more time to catch up.

Reply:
>  but two times may be too strict?

On the one hand, write operation is expensive, two times is not as small as it seems. On the other hand, not all but half hot fsms are rescheduled, that makes balance between threads as soon as possible.

> It may lead to a follower who falls behind its leader a lot uses much more time to catch up.

It's expected. This is the choice between fairness and efficiency.